modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryStorageEngine.java

```diff
@@ -106,27 +110,29 @@ public void start() throws StorageException {
                     ? new AsyncFileIoFactory()
                     : new RandomAccessFileIoFactory();
 
-            partitionFilePageStoreManager = new PartitionFilePageStoreManager(
-                    Loggers.forClass(PartitionFilePageStoreManager.class),
+            filePageStoreManager = new FilePageStoreManager(
```

Comment:
Can these fields be instantiated in constructor? Then they wouldn't have to be nullable

Reply:
Tell me which fields exactly.

Reply:
filePageStoreManager, meta manager as well. Fields that you instantiate in "start()"

Reply:
I don't understand what you're suggesting, please show me.

Reply:
Decided to leave it as it is for now.